internal/transform/trustline.go

```diff
@@ -61,7 +61,7 @@ func TransformTrustline(ledgerChange ingest.Change, header xdr.LedgerHeaderHisto
 	transformedTrustline := TrustlineOutput{
 		LedgerKey:          outputLedgerKey,
 		AccountID:          outputAccountID,
-		AssetType:          int32(asset.Type),
+		AssetType:          asset.Type.String(),
```

Comment:
nit: I think this will work but it would be nice to make the asset information consistent between the different transformations like [here](https://github.com/stellar/stellar-etl/blob/master/internal/transform/asset.go#L57-L60)
```
err := asset.Extract(&outputAssetType, &outputAssetCode, &outputAssetIssuer)
	if err != nil {
		return TrustlineOutput{}, fmt.Errorf("some error message")
	}
```

At least I think this code should have access to `Extract()`

Reply:
@chowbao Is there a reason why we are extracting assetType, code and issuer conditionally here:
https://github.com/stellar/stellar-etl/blob/0e50a2281af62fa67526bd7330557c72bce24f90/internal/transform/trustline.go#L44-L48

Can we extract this info regardless of assetType being poolshare?

Reply:
Oh oops the code already does this `Extract()`. Looking at your code it does seem like the way you wrote it is the easiest way to do this. `assetType` isn't even used in the code 😅 

The condition is needed because `xdr.AssetTypeAssetTypePoolShare` does not contain `assetCode` nor `assetIssuer`